engine/core/pom.xml

```diff
@@ -21,6 +21,20 @@
 		<dependency>
 			<groupId>org.apache.metamodel</groupId>
 			<artifactId>MetaModel-hbase</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>com.codehaus.jackson</groupId>
+					<artifactId>jackson-core-asl</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>com.codehaus.jackson</groupId>
+					<artifactId>jackson-mapper-asl</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>commons-logging</groupId>
+					<artifactId>commmons.logging</artifactId>
+				</exclusion>
+			</exclusions>
```

Comment:
I think there is confusion here... We did not add those exclusions, the team says. I can remove them. We think they are not needed.